src/include/funcapi.h

```diff
@@ -290,10 +290,18 @@ extern Datum HeapTupleHeaderGetDatum(HeapTupleHeader tuple);
 /* from funcapi.c */
 extern FuncCallContext *init_MultiFuncCall(PG_FUNCTION_ARGS);
 extern FuncCallContext *per_MultiFuncCall(PG_FUNCTION_ARGS);
+extern bool is_SquelchFuncCall(PG_FUNCTION_ARGS);
 extern void end_MultiFuncCall(PG_FUNCTION_ARGS, FuncCallContext *funcctx);
 
 #define SRF_IS_FIRSTCALL() (fcinfo->flinfo->fn_extra == NULL)
 
+/*
+ *	Set squelch flag for this function and test if it's squelching already
+ *  Note, that during squelch call function arguments are inaccessible,
+ *  so this call must appear before PG_GETARG_XXX() macros
```

Comment:
```suggestion
 * Set squelch flag for this function and test if it's squelching already.
 * Note, that during squelch call function arguments are inaccessible,
 * so this call must appear before PG_GETARG_XXX() macros
```

Reply:
pgindent reverts this comment to this format with tab and extra spaces. Personally I don't like it too, but who am I to argue with mighty pgindent.